atuin/src/command/client/search/interactive.rs

```diff
@@ -718,31 +727,16 @@ impl Stdout {
             event::EnableBracketedPaste,
         )?;
 
-        Ok(Self {
-            stdout,
-            inline_mode,
-        })
-    }
-
-    #[cfg(not(target_os = "windows"))]
-    pub fn new(inline_mode: bool) -> std::io::Result<Self> {
-        terminal::enable_raw_mode()?;
-        let mut stdout = stdout();
-
-        if !inline_mode {
-            execute!(stdout, terminal::EnterAlternateScreen)?;
+        if cfg!(not(target_os = "windows")) {
+            execute!(
+                stdout,
+                PushKeyboardEnhancementFlags(
+                    KeyboardEnhancementFlags::DISAMBIGUATE_ESCAPE_CODES
+                        | KeyboardEnhancementFlags::REPORT_ALL_KEYS_AS_ESCAPE_CODES
+                ),
+            )?;
```

Comment:
Could you use `#[cfg(not(windows))]` instead `cfg!`? Not too important but removes the code on windows instead of during runtime

Reply:
Ah, maybe that fails to compile in Windows?

I expected it to be optimized out in Windows even if `if cfg!(...)` is used, so the code will be removed in Windows anyway. The essential difference is that the semantic analysis such as type checking is skipped with `#[cfg(not(windows))]`, so `#[cfg(not(windows))]` can be used when the code inside doesn't compile in Windows.

Anyway, I don't have a strong preference, so I can change it.

Reply:
amended & rebased.